cibuildwheel/macos.py

```diff
@@ -81,6 +81,8 @@ def call(args, env=None, cwd=None, shell=False):
             env['PATH'],
         ])
         env = environment.as_dictionary(prev_environment=env)
+        if "MACOSX_DEPLOYMENT_TARGET" not in env:
+            env["MACOSX_DEPLOYMENT_TARGET"] = "10.9"
```

Comment:
If targeting macOS 10.9, then I see 2 options:
- keep building using 10.6 installer of python (32/64 bits)
- use the 10.9 installer of python when available (64 bits), dropping 32 bits support

Any idea what shall be done ? (Don't know if the analysis can be done to differentiate downloads triggered by 32/64 bits python)

Reply:
That's a good point, actually. So we're losing 32-bit compatibility?
But then, if I interpret [this table](https://en.wikipedia.org/wiki/MacOS#Hardware_compatibility) correctly, all versions after 10.6 were 64-bit only?

Reply:
With the PR in its current state, we don't loose 32-bit compatibility. It's also still possible to target 10.6 by defining `MACOSX_DEPLOYMENT_TARGET` to `10.6`.

10.6 installers do install a dual 32/64 bits python and, when building using these, wheels will be 32/64 bits. If switching to 10.9 installers then, wheels will only be 64 bits (and tags will need to be updated from `intel` to `x86_64`). As you mentioned, 10.7+ is always 64 bits (but can run 32bits software up to 10.14, 10.15 will drop 32 bits support). The PR in this state still builds 32/64 bits wheels (`intel` wheels) and is still the most compatible with everything out there. My question is : shall 32 bits support be dropped as well or not ? (by using 10.9 installers).

Reply:
There are few problems. Python 3.5 do not have 10.9 installer. So we cannot change installer in this case which introduce incompatibility. Maybe wait with changing installers to drop of Python3.5 support. 

What will be profit of updating versions?

Reply:
> What will be profit of updating versions?

Mostly:
- reducing build time
- reducing test time (well, since 32bits is probably not tested in most cases - at least all working examples, including mine-, probably not applicable)
- reducing package size

This being said, waiting for python 3.5 support to be dropped might be a good idea for consistency across python versions.

Reply:
Maybe add some MacOSX specific variable to allow user set other python installer?

Reply:
c.f. #202 / #203 for my point on 32-bit testing